Makefile

```diff
@@ -723,6 +723,8 @@ deploy/service: OBSERVABILITY_CONFIG_TAG ?= "main"
 deploy/service: DATAPLANE_CLUSTER_SCALING_TYPE ?= "manual"
 deploy/service: STRIMZI_OPERATOR_ADDON_ID ?= "managed-kafka-qe"
 deploy/service: KAS_FLEETSHARD_ADDON_ID ?= "kas-fleetshard-operator-qe"
+deploy/service: STRIMZI_OLM_PACKAGE_NAME ?= "managed-kafka"
+deploy/service: KAS_FLEETSHARD_OLM_PACKAGE_NAME ?= "kas-fleetshard-operator"
```

Comment:
Can you add a description for these 2 fields in the docs as well please :) https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager

Reply:
Make sense. I'll mark them as optional.